### PR TITLE
Design-23: New trash icons

### PR DESCRIPTION
**Pull request by isabelmachado**

### CODE REVIEW

File: nuxeo-icons.html [hunk 1]
```diff
@@ -64,8 +64,8 @@
     </g>
 
     <g id="attachment">
-      <path d="M2 12.5C2 9.46 4.46 7 7.5 7H18c2.21 0 4 1.79 4 4s-1.79 4-4 4H9.5C8.12 15 7 13.88 7 12.5S8.12 10 9.5 10H17v2H9.41c-.55 0-.55 1 0 1H18c1.1 0 2-.9 2-2s-.9-2-2-2H7.5C5.57 9 4 10.57 4 12.5S5.57 16 7.5 16H17v2H7.5C4.46 18 2 15.54 2 12.5z"></path>
-    </g>
+     <path d="M2 12.5C2 9.46 4.46 7 7.5 7H18c2.21 0 4 1.79 4 4s-1.79 4-4 4H9.5C8.12 15 7 13.88 7 12.5S8.12 10 9.5 10H17v2H9.41c-.55 0-.55 1 0 1H18c1.1 0 2-.9 2-2s-.9-2-2-2H7.5C5.57 9 4 10.57 4 12.5S5.57 16 7.5 16H17v2H7.5C4.46 18 2 15.54 2 12.5z"></path>
```
Comment (guirenard) [inline nuxeo-icons.html]: bad indentation

File: nuxeo-icons.html [hunk 2]
```diff
@@ -218,6 +222,10 @@
       <path d="M9.898,8l-2.949,-3l2.949,-3l-1.949,-1.949l-3,2.949l-3,-3l-1.949,2l2.949,3l-2.949,3l1.949,1.949l3,-2.949l3,2.949Z" transform="translate(7.051, 7.002)"></path>
     </g>
 
+    <g id="restore-deleted" fill-rule="nonzero">
+          <path d="M16,12.5 C16,11.2108333 14.9771429,10.1666667 13.7142857,10.1666667 C12.4514286,10.1666667 11.4285714,11.2108333 11.4285714,12.5 C11.4285714,13.7891667 12.4514286,14.8333333 13.7142857,14.8333333 C14.9771429,14.8333333 16,13.7891667 16,12.5 Z M13.7142857,2 C8.03428571,2 3.42857143,6.70166667 3.42857143,12.5 L0,12.5 L4.57142857,17.1666667 L9.14285714,12.5 L5.71428571,12.5 C5.71428571,7.99083333 9.29714286,4.33333333 13.7142857,4.33333333 C18.1314286,4.33333333 21.7142857,7.99083333 21.7142857,12.5 C21.7142857,17.0091667 18.1314286,20.6666667 13.7142857,20.6666667 C11.9828571,20.6666667 10.3885714,20.1008333 9.07428571,19.1441667 L7.45714286,20.8183333 C9.19428571,22.1833333 11.36,23 13.7142857,23 C19.3942857,23 24,18.2983333 24,12.5 C24,6.70166667 19.3942857,2 13.7142857,2 Z"></path>
+      </g>
```
Comment (guirenard) [inline nuxeo-icons.html]: idem

**Comment by guirenard** [comment]
:+1: but fix indentation please

**Comment by Gabez0r** [comment]
👍 to merge; last commit message is missing the ticket id, but you can squash the commits and keep the first message

**Comment by guirenard** [comment]
:+1:

**Pull request merged**